docs/_components/image-example.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Image Example
+lead: An image example displays an image with a do/don't header to help users upload  images correctly.
+---
+
+This is not a component in the USWDS design system.
+
+## Image Example Do
+
+
+{% capture example %}
+<div class="image-example image-example--correct">
```

Comment:
Keeping it consistent makes sense to me, especially since that's what we did for the other custom components. I prepended `usa-` to all the style and file names ([commit](https://github.com/18F/identity-design-system/pull/369/commits/51100fca9ed723957325dcc1e7bb3947b93818ea)).